docs/allocators.org

```diff
@@ -346,9 +348,16 @@ This pattern is pretty useful in restricted memory environments such as embedded
 ** TODO C allocator
 The ~C~ standard allocator can also be used, this allocator has high performance but it has less safety feature.
 
+** TODO implementing your own allocator
```

Comment:
removed it, when I wrote it I just thought that it would be cool to show that everyone can implement its own allocator, but after a bit of thinking I cant find any use case when any of the currently implemented allocators arent a good fit